src/machi.proto

```diff
@@ -424,6 +426,20 @@ message Mpb_LL_ReadChunkResp {
     repeated Mpb_ChunkPos trimmed = 3;
 }
 
+// Low level API: trim_chunk()
+
+message Mpb_LL_TrimChunkReq {
+    required Mpb_EpochID epoch_id = 1;
+    required string file = 2;
+    required uint64 offset = 3;
+    required uint32 size = 4;
+    optional uint32 trigger_gc = 5 [default=1];
+}
```

Comment:
Sorry, the default was wrong. Should be false at least. I have been wondering when and how to trigger GC in file proxy and it would be best if applications could control it. Background manager would be great and needed, but periodically circling around and trying GC could cause unnecessary read.

"Smart" application that has notion of files may know whether that file is still needed or no more needed ... then it can explicitly order flu servers to unlink the file, which is the most efficient.

Fortunately this option is not currently used and it's one of my TODOs. I'd love to discuss more.